peekingduck/cli.py

```diff
@@ -353,30 +356,3 @@ def _num_digits(number: int) -> int:
         (int): Number of digits in the given number.
     """
     return int(math.log10(number))
-
-
-def _verify_option(value: Optional[str], value_proc: Callable) -> Optional[str]:
-    """Verifies that input value via click.option matches the expected value.
-
-    This sets ``value`` to ``None`` if it is invalid so the rest of the prompt
-    can flow smoothly.
-
-    Args:
-        value (Optional[str]): Input value.
-        value_proc (Callable): A function to check the validity of ``value``.
-
-    Returns:
-        (Optional[str]): ``value`` if it is a valid value. ``None`` if it is
-            not.
-
-    Raises:
-        click.exceptions.UsageError: When ``value`` is invalid.
-    """
-    if value is None:
-        return value
-    try:
-        value = value_proc(value)
-    except click.exceptions.UsageError as error:
-        click.echo(f"Error: {error.message}", err=True)
-        value = None
-    return value
```

Comment:
We should update RTD with this new approach. Some things to consider:
1) Should this approach be the suggested "main" approach over the interactive method?
2) The readme to create custom nodes is becoming very long and could look intimidating. Could we move the "alternative" methods elsewhere, so that the user doesn't have to scroll so much between Steps 3 and 4?

Reply:
1. In the interest of letting the user get a custom node working as quickly as possible, `peekingduck create-node --config_path <path>` is suggested by default in Step 3.
2. The original section on using `peekingduck create-node` interactively is move to the bottom of the guide as a "bonus" section.

In the current section 3, some pros and cons of `peekingduck create-node --config_path <path>`  is discussed:
- pros: create lots of nodes quickly
- cons: skips on error, doesn't give a chance to fix

The "bonus" section is then recommended to the user if they want to run `create-node` interactively to be prompted to fix the errors